TESTNET.md

```diff
@@ -0,0 +1,36 @@
+# How to set up a corda k8s node on the corda testnet network
```

Comment:
Please merge this file with CHECKLIST.md.

Reply:
done